pg-commands/_template.md

```diff
@@ -0,0 +1,152 @@
+# ORDER BY
+
+<!---
+General structure:
+* Definition of keyword or function
+* Time-series focused friendly intro
+* Simple examples 
+* More complex examples 
+-->
+
+By default, in PostgreSQL and therefore TimescaleDB, the results from a basic
+`SELECT` query are unordered. You must include the [`ORDER BY`][orderby-postgres]
+clause to guarantee the order of your results. When you use `ORDER BY`, you need
+to specify the columns to order the results on then, optionally, give the
+ordering direction, either ascending or descending. If you don't specify the
+order direction for your columns, the default ordering is ascending.
+
+When you query time-series data, you need ordered results. Time occurs linearly
+and by nature is chronological. When you analyze and understand time-series
+data, you need to retain that order on time. To effectively analyze time-series
+data in TimescaleDB, the `ORDER BY` clause is necessary.
+
+## Syntax
+Single column ordering: sort all rows of data based on the values in the
+specified column:
+
+```sql
+ORDER BY <column_name> <optional ASC or DESC>
+```
+
+Multiple column ordering: the results are sorted by the specified columns one
+after another, so order matters:
+
+```sql
+ORDER BY <column_name> <optional ASC or DESC>, <column_name> <optional ASC or DESC>, …
+```
+
+<highlight type="note">
+Sorting operations are costly if indexes do not exist for the exact columns and
+ordering specified by the query. If sorting by multiple columns proves to be slow,
+look at an EXPLAIN plan for `(Sort)` nodes which indicate sorting that is taking
+place after the data is retrieved.
+</highlight>
+
+## Examples
+For example, if you have this raw  data:
+
+```sql
+stock_data:
```

Comment:
Should we make the table and some of the data match somewhat exactly the data from the tutorial and link to the tutorial?

Reply:
(I think we probably should)